packages/internet-header/cypress/e2e/meta-navigation.cy.ts

```diff
@@ -61,13 +61,13 @@ describe('meta-navigation', () => {
 
     describe('meta: changes during runtime (false => true => false)', () => {
       it(`adds and removes meta navigation`, () => {
-        cy.changeArg('meta', false);
+        cy.changeArg('meta', 'false');
         cy.get('post-meta-navigation').should('not.exist');
 
         cy.changeArg('meta', true);
```

Comment:
should this not also be `cy.changeArg('meta', 'true')`, if on the other hand it must be `'false'` instead of `false`?